app/assets/javascripts/discourse/app/routes/app-route-map.js

```diff
@@ -27,27 +27,7 @@ export default function () {
   });
 
   this.route("discovery", { path: "/", resetNamespace: true }, function () {
-    // top
-    this.route("top");
```

Comment:
If we remove all these routes, won't a link handled by the javascript app break? I understand server side routes have a redirect but I'm wondering about a link in a post?

Reply:
Yes, we have to redirect those routes in client-side too 👍 
`/top/monthly` => `/top?period=monthly`